demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2990,6 +3045,21 @@ export class DemoMeetingApp
       return this.bbprocessor;
     }
 
+    if (videoFilter.startsWith('Background Replacement')) {
+      console.log("background replacement - create called from videoFilterToProcessor!")
```

Comment:
- nit: Use `this.log('background replacement ...')`.
- typo? `videoFilterToProcessor`

Reply:
updated